tasks/system/Linux.yml

```diff
@@ -1,27 +1,66 @@
 ---
-- name: "Install contrib hybris components"
+- name: 'Install pip package'
+  package:
+    name: python-pip
+    state: present
+  register: status
+  until: status is succeeded
+  become: True
+  when: transport == 's3'
+
+- name: Install boto module
```

Comment:
These pre-requisites should be moved to pre_tasks or to prepare stage. It's not scope of hybris-config role to install boto.